appscale/tools/appscale.py

```diff
@@ -257,6 +259,12 @@ def up(self):
 
     # Construct a run-instances command from the file's contents
     command = []
+    if update:
+        command.append("--update")
+        update_dirs = update.split(',')
+        for dir in update_dirs:
```

Comment:
`dir` is a keyword in Python, so it's usually best to avoid using it as a variable name.

To simplify things a bit, you could just do `command.extend(update)` (assuming `update` is an iterable).